app/src/main/java/com/standuptracker/ui/home/HomeViewModel.kt

```diff
@@ -9,5 +9,6 @@ class HomeViewModel : ViewModel() {
     private val _text = MutableLiveData<String>().apply {
```

Comment:
Here you initialize a private variable and then make it public without any sort of safeguarding. Is there a reason for this encapsulation? You might consider just exposing the initial property as `internal` instead